creator-node/scripts/run-tests.sh

```diff
@@ -106,6 +106,8 @@ mkdir -p $storagePath
 # linter
 npm run lint
 
+export devMode=true
+
```

Comment:
surprised we havne't always been running tests with devMode = true

Reply:
this should be set here https://github.com/AudiusProject/audius-protocol/blob/4c559fd7b2a7e3aa5e4772c5b153260841fe9072/creator-node/compose/env/commonEnv.sh#L2 via https://github.com/AudiusProject/audius-protocol/blob/7eba9e5c57ea23ab5cb1fcd0fe7a54f22e113f86/creator-node/compose/docker-compose.yml#L30

Reply:
i intentionally only did it in run-tests bc i didn't want to modify all local dev without further testing
have a card to do that here https://linear.app/audius/issue/CON-252/improve-local-system-test-coverage
lmk thoughts

Reply:
@SidSethi sorry what i meant was doesn't it already have devMode set to true because of what's in commonEnv.sh?